frontend/lib/pages/data-driven-onboarding.tsx

```diff
@@ -342,9 +348,13 @@ function compareActionCardProps(a: ActionCardProps, b: ActionCardProps): number
 }
 
 function getSortedActionCards(data: DDOData): { recommended: ActionCardProps[], other: ActionCardProps[] } {
-  const actionCardProps = ACTION_CARDS.map(propsCreator => propsCreator(data)).map(props => (
-    props.imageStaticURL ? props : {...props, imageStaticURL: SHOW_PLACEHOLDER_IMG ? PLACEHOLDER_IMG : undefined}
-  ));
+  const actionCardProps = ACTION_CARDS
+    .map(propsCreator => propsCreator(data))
+      .map(props => {
+        if (DISABLE_RECOMMENDATIONS) { props.isRecommended = false; }
+        return (props.imageStaticURL ? props : {...props, imageStaticURL: SHOW_PLACEHOLDER_IMG ? PLACEHOLDER_IMG : undefined});
```

Comment:
Hmm, this feels a bit hard to read but I'm not sure of any particularly good ways to make it more readable.  One way might be to convert it all to something that uses spread notation like:

```js
return {
  ...props,
  isRecommended: DISABLE_RECOMMENDATIONS ? false : props.isRecommended,
  imageStaticURL: props.imageStaticURL || (SHOW_PLACEHOLDER_IMG ? PLACEHOLDER_IMG : undefined),
};
```

Uhh I'm not even sure if that's logically equivalent though, my brain hurts now.  Don't worry about it I guess.